src/server/mod.rs

```diff
@@ -12,7 +12,7 @@ use ::api::SensorTemplate;
 use ::datastore;
 use ::sensors;
 
-mod handlers;
+pub mod handlers;
```

Comment:
The end user doesn't even have access to the server module, see [src/lib.rs](https://github.com/coredump-ch/spaceapi-server-rs/pull/8/files#diff-b4aea3e418ccdb71239b96952d9cddb6L16) only `server::SpaceapiServer` and server::handlers::{StatusModifier, StateromPeopleNOwPresent} get exported.
Anyway, it's cleaner with a separate `modifiers` module so I'll go with that.